packaging/systemd/ncr.service

```diff
@@ -5,5 +5,9 @@ After=network.target
 [Service]
 ExecStart=/opt/ncr/ncr --config /etc/ncr/ncr.yml
 KillMode=process
-Restart=on-failure
+Restart=always
 User=ncr
+
+[Install]
+WantedBy=multi-user.target
+Alias=ncr.service
```

Comment:
The unit is already called `ncr`, so this alias seems pointless to me. Am I missing something?